osu.Framework/Input/FrameworkActionContainer.cs

```diff
@@ -15,6 +15,7 @@ internal class FrameworkActionContainer : KeyBindingContainer<FrameworkAction>
             new KeyBinding(new[] { InputKey.Control, InputKey.F11 }, FrameworkAction.CycleFrameStatistics),
             new KeyBinding(new[] { InputKey.Control, InputKey.F10 }, FrameworkAction.ToggleLogOverlay),
             new KeyBinding(new[] { InputKey.Alt, InputKey.Enter }, FrameworkAction.ToggleFullscreen),
+            new KeyBinding(new[] { InputKey.F11 }, FrameworkAction.ToggleFullscreen)
```

Comment:
unnecesssary array creation
```suggestion
            new KeyBinding(InputKey.F11, FrameworkAction.ToggleFullscreen)
```

Reply:
Array is there for consistency, and is only ever executed once in the lifetime of a host.  I don't see any problem in leaving it.